tests/includes/Helpers/DebugHelperTest.php

```diff
@@ -19,7 +19,7 @@ class DebugHelperTest extends PHPUnit_Framework_TestCase
 
     public function setUp()
     {
-        $this->dbh = $this->getMock(DebugHelper::class, array("get_debug_backtrace"));
+        $this->dbh = $this->getMockBuilder(DebugHelper::class)->setMethods(["get_debug_backtrace"])->getMock(); //(DebugHelper::class, array("get_debug_backtrace"));
```

Comment:
Don't comment out old code, just delete it. We have git if we want to look at old code 😄